Userland/Libraries/LibAudio/Loader.h

```diff
@@ -20,8 +20,8 @@
 
 namespace Audio {
 
-static const String empty_string = "";
-static String no_plugin_error = "No loader plugin available";
+static constexpr StringView empty_string = "";
```

Comment:
There should be an empty String or StringView available as a static member on one of them.

Reply:
Turns out, this is unused so I removed it.